### PR TITLE
BRE-380: Configure drone and remove travis

### PR DESCRIPTION
**Pull request by tylerlevine**
Signed-off-by: Tyler Levine <tyler@bitgo.com>

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+  - npm run test-node
+  environment:
+    BITGOJS_TEST_PASSWORD:
+      from_secret: password
+
+- name: upload coverage
+  image: node:11
+  commands:
+  - "npx nyc report --reporter=text-lcov > coverage.lcov"
+  - "npx codecov -f coverage.lcov -B \"${DRONE_BUILD_NUMBER:-0}\" -t \"$CODECOV_TOKEN\""
```
Comment (drGrove) [inline .drone.yml]: Shouldn't codecov just pull this from the environment? Do you have to be explicit?

File: .drone.jsonnet [hunk 2]
```diff
@@ -0,0 +1,106 @@
+local BuildInfo(version) = {
```
Comment (drGrove) [inline .drone.jsonnet]: Can you explain the need for this file? I see you have a .drone.yaml below so i'm not sure what this does

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+  commands:
+  - npm run test-node
+  environment:
+    BITGOJS_TEST_PASSWORD:
+      from_secret: password
+
+- name: upload coverage
+  image: node:9
+  commands:
+  - "npx nyc report --reporter=text-lcov > coverage.lcov"
```
Comment (drGrove) [inline .drone.yml]: Commands separated by - shouldn't need to be wrapped in strings

File: .drone.jsonnet (hunk 2 above)
Comment (tylerlevine) [reply .drone.jsonnet]: The `.drone.yml` file that you see in this PR was auto generated from this `.drone.jsonnet` file using `drone jsonnet --stream`.

File: .drone.yml (hunk 1 above)
Comment (tylerlevine) [reply .drone.yml]: I seem to remember that this only gets set automatically for a few CI providers, and drone was not one of them. However, I'm having trouble finding that page again now. I'll take a look at it, as I'd like to add flags as well.

**Comment by james8nguyen** [comment]
Just for the record, I had reviewed and approved the PR but did not do it fast enough. Got busy on some stuffs. Thanks.

**Pull request merged**